src/Zed/Dependency/Bridge/BridgeConstructorArgumentsRule.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace ArchitectureSniffer\Zed\Dependency\Bridge;
+
+use PDepend\Source\AST\ASTParameter;
+use PHPMD\AbstractNode;
+use PHPMD\Node\MethodNode;
+use PHPMD\Rule\MethodAware;
+
+/**
+ * Every bridge should not have type in constructor
+ */
+class BridgeConstructorArgumentsRule extends AbstractBridgeRule implements MethodAware
+{
+
+    /**
+     * @param \PHPMD\AbstractNode $node
+     *
+     * @return void
+     */
+    public function apply(AbstractNode $node)
+    {
+        if (!$this->isBridge($node)) {
+            return;
+        }
+
+        $this->applyRule($node);
+    }
+
+    /**
+     * @param \PHPMD\Node\MethodNode $method
+     *
+     * @return void
+     */
+    private function applyRule(MethodNode $method)
+    {
+        if ($method->getName() !== '__construct') {
+            return;
+        }
+
+        $params = $method->getParameters();
```

Comment:
This one should be extracted in a separate rule. It's a violation of SRP. Every Rule should only contain one check.